src/main/scala/com/softwaremill/reactive/complete/LargestDelayActorComplete.scala

```diff
@@ -16,6 +16,18 @@ class LargestDelayActorComplete extends PersistentActor with ActorSubscriber wit
     override def inFlightInternally = inFlight
   }
 
+  override def receive = {
```

Comment:
Hi, at least for me it didn't work when i 
started com.softwaremill.reactive.complete.ClusteredReceiver1
started com.softwaremill.reactive.complete.ClusteredReceiver2
started com.softwaremill.reactive.complete.ClusteredReceiver3
started com.softwaremill.reactive.complete.SenderComplete

all the messages were lost (dead letter)

*trying now without the method*
and it works,
damn for some reason it didn't work yesterday don't know why :/

Alright ill remove this